src/Moq/StringBuilderExtensions.AppendExpression.cs

```diff
@@ -327,39 +328,49 @@ private static StringBuilder AppendExpression(this StringBuilder builder, Method
 					builder.AppendNameOf(expression.Method.DeclaringType);
 				}
 
-				if (expression.Method.IsPropertyIndexerGetter())
+				if (expression.Method.IsGetAccessor())
 				{
-					builder.AppendCommaSeparated("[", expression.Arguments.Skip(paramFrom), AppendExpression, "]");
-				}
-				else if (expression.Method.IsPropertyIndexerSetter())
-				{
-					builder.AppendCommaSeparated("[", expression.Arguments.Skip(paramFrom).Take(expression.Arguments.Count - paramFrom - 1), AppendExpression, "] = ")
-					       .AppendExpression(expression.Arguments.Last());
-				}
-				else if (expression.Method.IsPropertyGetter())
-				{
-					builder.Append('.')
-					       .Append(expression.Method.Name, 4);
-					if (expression.Arguments.Count > paramFrom)
+					if (expression.Method.IsPropertyAccessor())
+					{
+						builder.Append('.')
+						       .Append(expression.Method.Name, 4);
+						if (expression.Arguments.Count > paramFrom)
+						{
+							builder.AppendCommaSeparated("[", expression.Arguments.Skip(paramFrom), AppendExpression, "]");
```

Comment:
Not introduced by this refactoring, but weird nonetheless: Why do angle brackets appear in the formatting of a property getter expression?